docs/core/actions.rst

```diff
@@ -144,6 +144,43 @@ There is an example of a ``SlotSet`` event
 :ref:`above <custom_action_example>`, and a full list of possible
 events in :ref:`Events <events>`.
 
+Execute Actions in Other Code
```

Comment:
moved this up. Makes more sense here imho.

Reply:
agreed